packages/astro/src/core/util.ts

```diff
@@ -97,6 +98,32 @@ export function viteID(filePath: URL): string {
 	return slash(fileURLToPath(filePath));
 }
 
+/** An fs utility, similar to `rimraf` or `rm -rf` */
+export function removeDir(_dir: URL): void {
+  const dir = fileURLToPath(_dir);
+	fs.rmSync(dir, {recursive: true, force: true});
+}
+
+export function emptyDir(_dir: URL, skip?: Set<string>): void {
+  const dir = fileURLToPath(_dir);
+	for (const file of fs.readdirSync(dir)) {		
```

Comment:
Looks like this needs an `fs.existsSync(dir)` check first, I never understood why `fs.read*` APIs don't just return an empty array if the directory isn't found 🤷